api_auth/User/serializers.py

```diff
@@ -49,3 +49,18 @@ def validate_phone_number(self, value):
             raise serializers.ValidationError('Phone number cannot be empty')
 
         return value
+
+class UserPasswordSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ('password')
+
+    def validate_password(self, value):
```

Comment:
Django has a builtin validate_password that we can configure with the desired rules. So, I guess we don't need to manually implement it. Check this for further info on that: [Django Documentation](https://docs.djangoproject.com/en/5.0/topics/auth/passwords/#module-django.contrib.auth.password_validation)